mevolib/cluster/Genes.py

```diff
@@ -125,215 +85,171 @@
                     '5\'UTR': ['gene', 'standard_name'] }
 
 
-#-------------------------------------------------------------------------------
-
-def _normalization ( record, refseq_record, alignment_bin ) :
-    """
-    Normalization of the input sequence with the reference sequence. The
-    normalization consists on aligning both sequences and removing those sites
-    where a gap has been introduced in the reference sequence, making the
-    features of the reference sequence applicable to the new sequence. It
-    returns the new sequence and the reference sequence's features.
+def _normalization(record: Bio.SeqRecord, refseq_record: Bio.SeqRecord, 
+                   alignment_bin: str) -> tuple(Bio.Seq, List):
+    """Normalization of the input sequence with the reference sequence. The normalization consists on aligning 
+    both sequences and removing those sites where a gap has been introduced in the reference sequence, making 
+    the features of the reference sequence applicable to the new sequence. It returns the new sequence and the 
+    reference sequence's features.
 
-    Arguments :
-        record  ( Bio.SeqRecord )
-            Sequence to normalize.
-        refseq_record  ( Bio.SeqRecord )
-            Reference sequence of the same type as 'record'.
-        alignment_bin  ( string )
-            Binary path to the alignment tool that will be used in the
-            normalization process.
+    Args:
+        record: Sequence to normalize.
+        refseq_record: Reference sequence of the same type as 'record'.
```

Comment:
To highlight arguments we use "`":
```suggestion
        refseq_record: Reference sequence of the same type as `record`.
```